src/gpu/intel/gemm/jit/generator_dsl/builder.cpp

```diff
@@ -228,9 +226,9 @@ void apply_post_ops(const dnnl::impl::gpu::intel::gpu_post_ops_t &ops,
         const std::vector<idx_t> &dims) {
     for (size_t i = 0; i < ops.len(); i++) {
         if (ops[i].is_eltwise()) {
-            gpu_assert(false) << "Unimplemeted";
+            dsl_assert(false) << "Unimplemeted";
```

Comment:
```suggestion
            stub();
```

And a similar line below.